libs/utils/utils.js

```diff
@@ -282,14 +282,28 @@ export const loadScript = (url, type) => new Promise((resolve, reject) => {
     if (type) {
       script.setAttribute('type', type);
     }
-    script.onload = () => {
-      resolve(script);
-    };
-    script.onerror = () => {
-      reject(new Error('error loading script'));
-    };
     head.append(script);
   }
+
```

Comment:
@Brandon32 What's your opinion here, since you're the most familiar with this

Reply:
Looking at it again I realized that on the second call the event listeners do need to be added if the dataset.loaded isn't true, otherwise the call will never resolve the script.  I've updated the code.

Reply:
Updated